bindings/time-profiler.cc

```diff
@@ -19,7 +19,7 @@
 
 using namespace v8;
 
-Local<Value> TranslateCpuProfileNode(const CpuProfileNode* node) {
+Local<Value> TranslateTimeProfileNode(const CpuProfileNode* node) {
   // TODO: Implement unimplemented interface
```

Comment:
This (and below) comment is not super-clear: which interface? Could you clarify?

Reply:
@ofrobots -- I think this comment is from the original code. Any ideas what it refers to?

Reply:
I am not quite sure. Feel free to remove it.